src/indev/lv_indev.c

```diff
@@ -1629,19 +1629,33 @@ static void indev_reset_core(lv_indev_t * indev, lv_obj_t * obj)
 
 static lv_result_t send_event(lv_event_code_t code, void * param)
 {
-    lv_obj_send_event(indev_obj_act, code, param);
-    if(indev_reset_check(indev_act)) return LV_RESULT_INVALID;
+    lv_indev_t * indev = indev_act;
 
     if(code == LV_EVENT_PRESSED ||
+       code == LV_EVENT_SHORT_CLICKED ||
        code == LV_EVENT_CLICKED ||
        code == LV_EVENT_RELEASED ||
        code == LV_EVENT_LONG_PRESSED ||
        code == LV_EVENT_LONG_PRESSED_REPEAT ||
        code == LV_EVENT_ROTARY) {
-        lv_indev_send_event(indev_act, code, indev_obj_act);
-        if(indev_reset_check(indev_act)) return LV_RESULT_INVALID;
+        lv_indev_send_event(indev, code, indev_obj_act);
+        if(indev_reset_check(indev)) return LV_RESULT_INVALID;
+
+        /* Not send click or long pressed event if scroll is detected */
+        if(code == LV_EVENT_SHORT_CLICKED ||
+           code == LV_EVENT_CLICKED ||
+           code == LV_EVENT_LONG_PRESSED ||
+           code == LV_EVENT_LONG_PRESSED_REPEAT) {
+            if(LV_ABS(indev->pointer.scroll_sum.x >= indev->scroll_limit) ||
+               LV_ABS(indev->pointer.scroll_sum.y >= indev->scroll_limit)) {
+                return LV_RESULT_OK;
+            }
+        }
```

Comment:
@kisvegabor This is a modification of the scroll judgment

Reply:
Oh, I should have put my glasses on. :sweat_smile: Thanks!